src/ElasticSearchIndexer.php

```diff
@@ -214,4 +220,15 @@ private function extendBatch(string $action, string $indexName, IndexDocument $d
 
         return $params;
     }
+
+    private function checkErrors(array $clientResponse): string
+    {
+        $errors = '';
+        
+        foreach ($clientResponse['items'] as $response) {
+            $response = reset($response);
+            $errors .= $response['error']['reason'];
+        }
+        return $errors;
+    }
```

Comment:
I think we can rename this method, because I didn't see any `error checks` here.
Maybe something like `getErrors`, `parseErrors`, `buildErrors`, `extractErrors`, etc.
Wdyt?